src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
 body {
   margin: 0;
+  background-color: gray;
```

Comment:
The background color should be according to the mockup provided. If the mockup does not provide a background color, consider using a more neutral color or remove the background color.